src/views/partials/footer.hbs

```diff
@@ -0,0 +1,4 @@
+{{#each js}}
```

Comment:
that should be in ``` layout ``` itself not in ``` footer ``` partial  
i mean script tag

Reply:
done.